buildSrc/src/main/java/org/elasticsearch/gradle/Distribution.java

```diff
@@ -20,24 +20,26 @@
 
 public enum Distribution {
 
-    INTEG_TEST("elasticsearch", "integ-test-zip"),
-    DEFAULT("elasticsearch", "elasticsearch"),
-    OSS("elasticsearch-oss", "elasticsearch-oss");
+    INTEG_TEST("elasticsearch"),
+    DEFAULT("elasticsearch"),
+    OSS("elasticsearch-oss");
 
     private final String artifactName;
-    private final String group;
 
-    Distribution(String name, String group) {
+    Distribution(String name) {
         this.artifactName = name;
-        this.group = group;
     }
 
     public String getArtifactName() {
         return artifactName;
     }
 
     public String getGroup() {
-        return "org.elasticsearch.distribution." + group;
+        if (this.equals(INTEG_TEST)) {
```

Comment:
I kinda preferred this being explicit where we define the enum values rather than being conditional like this.